stats/stats_central.php

```diff
@@ -70,12 +84,8 @@
         ""
     );
 
-  //get total books posted  in the last 7 days
-
-    $books = mysqli_query(DPDatabase::get_connection(), "SELECT count(*) AS numbooks FROM projects
-                          WHERE modifieddate >= $begin_time AND state = '".PROJ_SUBMIT_PG_POSTED."'");
-    $row = mysqli_fetch_assoc($books);
-    $totalbooks = $row["numbooks"];
+    //get total books posted  in the last 7 days
+    $totalbooks = count_books_in_state(PROJ_SUBMIT_PG_POSTED, "AND modifieddate >= $begin_time");
```

Comment:
do we want to use sprintf here with %d?

Reply:
We could, but given that `$begin_time` doesn't depend on data under control of the user, I think we're pretty safe.

```$begin_time = time() - 604800; // in seconds```